scene/main/node.cpp

```diff
@@ -2856,6 +2858,7 @@ void Node::_bind_methods() {
 
 	BIND_CONSTANT(NOTIFICATION_ENTER_TREE);
 	BIND_CONSTANT(NOTIFICATION_EXIT_TREE);
+	BIND_CONSTANT(NOTIFICATION_PRE_EXIT_TREE);
```

Comment:
This should be added in the same order as it's included in the enum. You can see in the XML docs that the values are out of sync (30 between 11 and 12).